project3/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700|Teko:500,600,700&display=swap"
+        rel="stylesheet">
+    <title>SUPER HERO REGISTRY</title>
+</head>
+
+<body>
+
+    <form>
+
+        <fieldset>
+            <legend>Identity</legend>
+            <label for="real-name"> Real Name: </label>
+            <input id="real-name" type="text" name="realname" required>
+            <label for="alias">Alias:</label>
+            <input id="alias" type="text" name="alias" required>
+            <label for="email"> Email: </label>
+            <input id="email" type="text" name="email" required>
+        </fieldset>
+        <fieldset>
+            <legend>Personal Characteristics</legend>
+            <label for="age"> Age:</label>
+            <input id="age" type="number" required>
+            <fieldset>
+                <legend>Category</legend>
+                <span>
+                    <input name="category" id="alien" type="checkbox">
+                    <label for="alien">Alien</label>
+                </span>
+                <span>
+                    <input name="category" id="cyborg" type="checkbox">
+                    <label for="cyborg">Cyborg</label>
+                </span>
+                <span>
+                    <input name="category" id="experiment" type="checkbox">
+                    <label for="experiment">Experiment</label>
+                </span>
+                <span>
+                    <input name="category" id="human" type="checkbox">
+                    <label for="human">Human</label>
+                </span>
+                <span>
+                    <input name="category" id="inhuman" type="checkbox">
+                    <label for="inhuman">Inhuman</label>
+                </span>
+                <span>
+                    <input name="category" id="mutant" type="checkbox">
+                    <label for="mutant"> Mutant</label>
+                </span>
+
+            </fieldset>
+
+            <fieldset>
+                <legend>Gender</legend>
```

Comment:
Per [requirements](https://mica-web.github.io/gd-220/proj3.html#requirements), the answer to gender should be **required**.